README.md

```diff
@@ -123,29 +128,20 @@ to run in-cluster.
 
 ## Deployment on cluster
 
-After pushing an image with your changes to a registry, you can deploy the
-plugin to a cluster by instantiating the provided
-[OpenShift template](template.yaml). It will run a light-weight nginx HTTP
-server to serve your plugin's assets.
-
-```sh
-oc process -f template.yaml \
-  -p PLUGIN_NAME=my-plugin \
-  -p NAMESPACE=my-plugin-namespace \
-  -p IMAGE=quay.io/my-repository/my-plugin:latest \
-  | oc create -f -
-```
+A [Helm](https://helm.sh) chart is available to deploy the plugin to an OpenShift environment.
+
+The following Helm parameters are required:
+
+`plugin.image`: The location of the image containing the plugin that was previously pushed
+
+Additional parameters can be specified if desired. Consult the chart [values](charts/openshift-console-plugin/values.yaml) file for the full set of supported parameters.
 
-`PLUGIN_NAME` must match the plugin name you used in the `consolePlugin`
-declaration of [package.json](package.json).
+### Installing the Helm Chart
 
-Once deployed, patch the
-[Console operator](https://github.com/openshift/console-operator)
-config to enable the plugin.
+Install the chart using the name of the plugin as the Helm release name into a new namespace or an existing namespace as specified by the `my-plugin-namespace` parameter by using the following command:
 
-```sh
-oc patch consoles.operator.openshift.io cluster \
-  --patch '{ "spec": { "plugins": ["my-plugin"] } }' --type=merge
+```shell
+helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace
```

Comment:
For this command to succeed, we need to supply an image here, correct? Is this typically done by editing values.yaml?

We probably want to make that a little more clear and give a recommendation (whether it's editing values.yaml or using `--set`).

Reply:
I did receive these warnings trying this due to the new pod security admission controller in 4.12.

```
W0825 08:02:06.016212   45304 warnings.go:70] would violate PodSecurity "restricted:v1.24": allowPrivilegeEscalation != false (container "console-plugin-template" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "console-plugin-template" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "console-plugin-template" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "console-plugin-template" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
W0825 08:02:06.330977   45304 warnings.go:70] would violate PodSecurity "restricted:v1.24": allowPrivilegeEscalation != false (container "console-plugin-template-patcher" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "console-plugin-template-patcher" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "console-plugin-template-patcher" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "console-plugin-template-patcher" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
```